Sorts/test/MergeSort.test.ts

```diff
@@ -0,0 +1,25 @@
+import { MergeSort } from "../MergeSort";
+
+describe("Merge Sort", () => {
+    it("generating array with length 10 and comparing with sorted array", () => {
```

Comment:
Please don't duplicate this code; rather use something like Jest's `each`, or write your own utility func to test a random sorted array of a given length.

Reply:
Let me know if the change works